runners/flink/src/main/java/org/apache/beam/runners/flink/TestFlinkRunner.java

```diff
@@ -55,7 +55,13 @@ public static TestFlinkRunner create(boolean streaming) {
   @Override
   public PipelineResult run(Pipeline pipeline) {
     try {
-      return delegate.run(pipeline);
+      PipelineResult job = delegate.run(pipeline);
+
+      job.waitUntilFinish();
+//      job.waitUntilFinish(
```

Comment:
Ah yes, I wanted to add this and actually make these tests `@ValidatesRunner` tests so that all runners are verified.